src/LoanActionsHistory.js

```diff
@@ -21,6 +21,7 @@ import {
   KeyValue,
   Row,
   Col,
+  IfPermission,
```

Comment:
`IfPermission` is deprecated in `stripes v1.1` and has moved to `stripes-core`. Otherwise, this looks good.